app/core/download.py

```diff
@@ -175,7 +172,10 @@ def create_query() -> str:
     n3.event_type_names as "Full timeline of events (types)",
     n3.event_dates as "Full timeline of events (dates)",
     d.created::date as "Date Added to System",
-    f.last_modified::date as " Last Modified on System"
+    f.last_modified::date as " Last Modified on System",
```

Comment:
Theres a bit of extra whitespace here 🔎

Reply:
No whitespace here - the line has been moved 😄

Reply:
Whoops - see what you mean. Have fixed.